src/orderbook.ts

```diff
@@ -0,0 +1,34 @@
+export class Orderbook {
+  from: string;
```

Comment:
nit: What is this from? By the example data in the test, it looks like it was used for the token symbol.
If it's just a label, maybe something like `sellLabel` or `sellTokenLabel`.
If it's an address, `sellTokenAddress`

Reply:
Agree that this naming is not very clear. However in an orderbook the notion of a sell/buy token does not really exist as the same token can be sold (to bidders) and bought (from askers).

I think the current name is base currency and quote currency (cf https://support.kraken.com/hc/en-us/articles/115000364388-Trading-glossary). Will change

Reply:
okay, look good to me if you change it to those names you suggest. 
Just not "from/to" cause it can be very confusing, specially in crypto.